src/undocumented_names.jl

```diff
@@ -0,0 +1,33 @@
+"""
+    test_undocumented_names(module::Module)
+
+Test that all public names in `module` have a docstring (not including the module itself).
+
+!!! tip
+    On all Julia versions, public names include the exported names.
+    On Julia versions >= 1.11, public names also include the names annotated with the `public` keyword.
+
+!!! warning
+    When running this Aqua test in Julia versions before 1.11, it does nothing.
+    Thus if you use continuous integration tests, make sure those are configured
+    to use Julia >= 1.11 in order to benefit from this test.
+"""
+function test_undocumented_names(m::Module)
```

Comment:
This should also accept a `broken` kwarg, with the same semantics as for e.g. `test_undefined_exports`

Reply:
done in the latest commit